limits.c

```diff
@@ -359,5 +368,7 @@ void limits_force_servo(){
   servo_line_number++; // increment for next time we peform this process
   
   plan_sync_position(); // Sync planner position to current machine position for pull-off move.
+
+  limits.mag_gap_check = 1; // Start checking magazine gaps on carousel again	
```

Comment:
This should probably be setting mag_gap_check equal to the MAG_GAP_CHECK_ENABLE define (so that this behavior is obeyed)